pydm/widgets/drawing.py

```diff
@@ -1121,60 +1126,56 @@ def p2d(pt):
     def getPoints(self):
         return self._points
 
-    def _validator_(self, value):
+    def _validator(self, value):
         """
         ensure that `value` has correct form
 
         Parameters
         ----------
-        value : [str]
+        value : [ordered pairs]
             List of strings representing ordered pairs
             of integer coordinates.  Each ordered pair
-            is comma-separated (although white-space
-            separated is acceptable as input).
+            is a tuple or list.
 
         Returns
         ----------
-        verified : [str]
-            List of strings in standard format
+        verified : [ordered pairs]
+            List of `tuple(number, number)`.
 
         """
-        def isinteger(value):
-            value = value.strip()
+        def isfloat(value):
+            if isinstance(value, str):
+                value = value.strip()
             try:
                 float(value)
                 return True
             except:
                 return False
 
         verified = []
-        for i, pt in enumerate(value):
-            point = pt.split(",")
-            if len(point) != 2:
-                point = pt.split()    # tolerant of space-separated
+        for i, pt in enumerate(value, start=1):
+            point = pt
+            if isinstance(point, str):
+                point = list(map(float, point.split(",")))
+                # logger.exception("polyline point %d must be list of (number, number) pairs, received '%s'", i, pt, exc_info=False)
+                # return
```

Comment:
Stale comment
```suggestion
```

Reply:
Good catch.  I'll resolve this from my session.